book-service-rest/src/main/java/com/example/Library/exceptions/UserAlreadyExists.java

```diff
@@ -0,0 +1,7 @@
+package com.example.Library.exceptions;
+
+public class UserAlreadyExists extends RuntimeException {
```

Comment:
do you have a proper exception handing for this Exception? It is a good practice to return 409 response code in case of such conflicts